sheepdog/transactions/upload/entity.py

```diff
@@ -534,81 +552,9 @@ def get_metadata(self):
         metadata = {'acls': flask.g.dbgap_accession_numbers}
         return metadata
```

Comment:
this should be moved the the file subclass and the metadata should be added to indexd call?